Default.preset

```diff
@@ -89,6 +91,9 @@ SetVisualFXPerformance
 
 ShowKnownExtensions
 ShowHiddenFiles
+# EnableFolderSeparateProcess

```

Comment:
The name is there always 3 times. One in preset file, one in default preset, one in the actual function. Rename needs to be done in all 3.